Merge the Andoid 9 Branch to Master

TechG3n opened the pull request.

TechG3n committed:
Move #39 and #40 to own branch, add a missing fi

Commit:
Add files via upload

Missing fi and another missing if version check for starting atlas at another point in the script.

TechG3n committed:
Merge pull request #42 from roundaboutluke/rom_A9

more a9 fixes

Cronick committed:
Typo-fix (verison -> version)

Cronick committed:
If statement indentation

Cronick committed:
Update README.md

TechG3n committed:
Merge pull request #45 from Cronick/rom_A9

Minor changes for A9

Commit:
Readme updated

Currently you need to manually insert aconf_versions into /data/local before running - this command pulls it down first while still maintaining the functionality of the original command.

Commit:
new command security fix

as discussed with TechG3n, on Pogo Tools Discord a security fix was pointed out by @mattle to ensure system is made read only again

TechG3n committed:
Merge pull request #46 from roundaboutluke/rom_A9

Readme updated - new launch command

TechG3n committed:
Make it possible to set a system proxy

TechG3n committed:
Small fix for md5 generation, thanks to snowbie

TechG3n committed:
Merge pull request #50 from TechG3n/rom_A9_fix1

Small fix for md5 generation, thanks to snowbie

TechG3n committed:
Change Readme to the new rom
Adjust ATVdetailsSender to A9

TechG3n committed:
version bumps

TechG3n committed:
Merge pull request #54 from TechG3n/rom_A9_fix2

Change Readme to the new rom

TechG3n committed:
Give ATVs a temp hostname for Atlas Dashboard

TechG3n committed:
Merge pull request #55 from TechG3n/rom_A9_fix3

Give ATVs a temp hostname for Atlas Dashboard

TechG3n committed:
Make ATVdetailsSender differ between Android versions

TechG3n committed:
Merge pull request #56 from TechG3n/rom_A9_fix4

Make ATVdetailsSender differ between Android versions

TechG3n committed:
correct some metrics

AppIe314 commented on scripts/ATVdetailsSender.sh:
```diff
@@ -50,15 +51,15 @@ while true
     memAtlas=$(dumpsys meminfo 'com.pokemod.atlas:mapping' | grep -m 1 "TOTAL" | awk '{print $2}')
-    cpuSys=$(top -n 1 | grep -m 1 "System" | awk '{print substr($2, 1, length($2)-2)}')
-    cpuUser=$(top -n 1 | grep -m 1 "User" | awk '{print substr($2, 1, length($2)-2)}')
+    #cpuSys=$(top -n 1 | grep %sys | awk 'NR == 1 {sub(/%sys/, "", $4); print $4}')
```
Is there a reason why these variables are commented out?  Could delete them instead.

AppIe314 commented on scripts/ATVdetailsSender.sh:
```diff
@@ -86,6 +87,24 @@ while true
+    cpuUser=$(top -n 1 | grep %user | awk 'NR == 1 {sub(/%user/, "", $2); print $2}')
+    cpuPogoPct=$(dumpsys cpuinfo | grep 'com.nianticlabs.pokemongo' | awk '{print substr($1, 1, length($1)-1)}')
+    cpuApct=$(dumpsys cpuinfo | grep 'com.pokemod.atlas' | awk 'NR == 1 {sub(/%/, "", $1); print $1}')
+    #Still not sure about these
```
Has anyone tested this yet? Maybe need to ask in PoGoRom Dev server.

TechG3n replied:
tested by me, feel free to test.

AppIe314 replied:
Working for me too on A9 S905W.

TechG3n committed:
As requested, set proxy only in same network as proxy. So remote ATVs aren't effected (if they don't use the same network)

TechG3n committed:
remove as requested

TechG3n committed:
remove discord msg

matt-yml committed:
freeze Werkzeug version at 2.3.7
* this is the latest version that aconf can be run with
* fixes Flask 2.1.1 runtime error:
`ImportError: cannot import name 'url_quote' from 'werkzeug.urls`

Flask and Werkzeug need to be kept in sync over major versions

TechG3n committed:
Merge pull request #58 from TechG3n/rom_A9_fix5

As requested, set proxy only in same network as proxy.

Cronick committed:
Update ATVdetailsSender.sh

TechG3n committed:
Merge pull request #60 from Cronick/rom_A9

Datafixes for A9

matt-yml committed:
Merge branch 'TechG3n:rom_A9' into rom_A9

matt-yml committed:
update additional packages per @techG3n github dependency bot
recommendations for security issues.

https://github.com/TechG3n/aconf/pull/59#issuecomment-1783787655

matt-yml committed:
Merge branch 'rom_A9' of github.com:matt-yml/aconf-dev into rom_A9

TechG3n committed:
Merge pull request #59 from matt-yml/rom_A9

freeze Werkzeug version at 2.3.7

TechG3n commented:
Have been running this on my mixed A7 / A9 setup for a while, no problems. Can merge this soon

colinchaput committed:
Fixed several errors and grammar mistakes. Fixes #62

TechG3n committed:
Merge branch 'rom_A9' into spelling-fixes-readme-#62

TechG3n committed:
Merge pull request #63 from colinchaput/spelling-fixes-readme-#62

Fixed several errors and grammar mistakes. Fixes #62

TechG3n committed:
Update wiki
